src/Microsoft.DocAsCode.Build.Engine/ManifestUtility.cs

```diff
@@ -26,8 +27,12 @@ public static void RemoveDuplicateOutputFiles(ManifestItemCollection manifestIte
                               .GroupBy(obj => obj.relativePath, FilePathComparer.OSPlatformSensitiveStringComparer)
                               .Where(g => g.Count() > 1))
             {
-                Logger.LogWarning($"Overwrite occurs while input files \"{string.Join(", ", duplicates.Select(duplicate => duplicate.item.SourceRelativePath))}\" writing to the same output file \"{duplicates.Key}\"");
-                itemsToRemove.UnionWith(duplicates.Skip(1).Select(duplicate => duplicate.item.SourceRelativePath));
+                var duplicateSources = duplicates.Skip(1).Select(duplicate => duplicate.item.SourceRelativePath).ToList();
+                // TODO: plan to change this warning to error, add error code to analyze the impact.
```

Comment:
what is the plan? even if it is an error, we should fail after all documents are built?